Problem1.py

```diff
@@ -0,0 +1,68 @@
+"""
+// Time Complexity : o(m*3)
```

Comment:
is it m power 3?

Reply:
No, its m multiplied by 3, where m is the number of houses and 3 for the colors.